cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -2538,21 +2536,19 @@ SwiftGenerator::MetadataVisitor::visitOperation(const OperationPtr& p)
         }
     }
     p->setMetadata(validate(p, metadata, p->file(), p->line()));
-    ParamDeclList params = p->parameters();
-    for (ParamDeclList::iterator q = params.begin(); q != params.end(); ++q)
+    for (auto& param : p->parameters())
```

Comment:
const auto&

Reply:
Changed.

I was just keeping the original semantics; we used `iterator` and not `const_iterator`,
probably because within the loop, we were calling `setMetadata` which mutates the `param` (but through a ptr).